lib/flutter_nfc_kit.dart

```diff
@@ -344,7 +376,7 @@ class FlutterNfcKit {
   }
 
   /// iOS only, change currently displayed NFC reader session alert message with [message].
-  /// 
+  ///
```

Comment:
If you want to format code, please move these changes to a separate pull request.